lcoe_calculator/base_processor.py

```diff
@@ -524,4 +528,31 @@ def _calc_lcoe(self):
                 self.df_aep.values)
         df_lcoe = df_lcoe + self.df_vom.values - ptc
 
-        return df_lcoe
\ No newline at end of file
+        return df_lcoe
+    
+    def _get_tax_credit_case(self):
+        if self.has_tax_credit:
+            assert len(self.df_tc) > 0, \
+                (f'Setup df_tc with extractor.get_tax_credits() before calling this function!')
+
+            ptc = self._calc_ptc()
+            itc = self._calc_itc()
+
+            # Trim the first year to eliminate pre-inflation reduction act confusion
```

Comment:
Is this "trim first year" or is it more of "trim year 20XX"? Will this still be valid next year?